services/metadata/src/metadata/services/avus.clj

```diff
@@ -55,7 +54,7 @@
       (transaction
        (persistence/remove-avu-template-instances template-id [avu-id])
        (persistence/remove-avu avu-id))
-      (throw+ {:error_code error-codes/ERR_DOES_NOT_EXIST
+      (throw+ {:type :clojure-commons.exception/exists
```

Comment:
(not sure if "not found" is equivalent. possibly/probably. but it's not this, anyway)